tests/test_reference_velocity_verlet_integrator.py

```diff
@@ -0,0 +1,123 @@
+from functools import partial
+
+import numpy as np
+from jax import config, grad, jit
+from jax import numpy as jnp
+
+config.update("jax_enable_x64", True)
+
+from timemachine.integrator import VelocityVerletIntegrator
+from timemachine.testsystems.relative import hif2a_ligand_pair
+
+
+def assert_reversible(x0, v0, update_fxn, atol=1e-10):
+    """Assert that
+    * I(x0, v0) != (x0, v0)
```

Comment:
`I` hear means integrate and `F` means flip?

Reply:
Correct

Reply:
Clarified this comment in https://github.com/proteneer/timemachine/pull/776/commits/5a2bdc9aa81837364aa18223d759b5fed01c7d82 and https://github.com/proteneer/timemachine/pull/776/commits/6eabb32f1fe60699d9a3a15e40ec33127262eaee